docs/developers_guide/building_docs.rst

```diff
@@ -17,4 +17,8 @@ Then, run the following script to build the docs:
     rm -rf developers_guide/generated/ developers_guide/*/generated/ _build/
     make html
 
+You may need to rerun your compass load script for the api docs to build
+successfully if you have added new modules since the load script was
+last run.
+
```

Comment:
An important addition.  Let me offer a couple of edits:
```suggestion
You may need to re-source your compass load script in the root of the compass branch
for the api docs to build successfully if you have added new modules since the load
script was last sourced.  The load script will reinstall compass into the conda
environment when it is sourced in the root of the compass branch.

```